R/render.r

```diff
@@ -482,7 +577,16 @@ check_bootswatch_theme <- function(bootswatch_theme, bs_version, pkg) {
 }
 
 pkgdown_bslib_defaults <- function() {
-  list(`navbar-nav-link-padding-x` = "1rem")
+  list(
+    `navbar-nav-link-padding-x` = "1rem",
+    `primary` = "#0054AD",
+    `secondary` = "#e9ecef",
+    `navbar-bg` = "#f8f9fa",
```

Comment:
instead maybe `navbar-bg` should be a mix of variables: probably `bg` and something else (`fg`? `secondary`? `primary`?)